### PR TITLE
Update hello-minikube.md for M1 mac notebooks

### PR DESCRIPTION
**Pull request by f4x15**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA

**Comment by f4x15** [comment]
Signed. Please check and merge

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -96,6 +96,11 @@ Pod runs a Container based on the provided Docker image.
     ```shell
     kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.4
     ```
+    
+For M1 laptops you could run:
```
Comment (sftim) [inline content/en/docs/tutorials/hello-minikube.md:100]: Thanks. We should specify a broader context; this applies to any computer that uses AArch64 CPUs and not just a specific kind of laptop.

We should _also_ fix the echoserver image; see https://github.com/kubernetes/website/pull/36731#issuecomment-1250246299 and other comments on that PR.

**Comment by mtardy** [comment]
This merged PR should solve the issue without adding the warning message https://github.com/kubernetes/website/pull/37383.

Feel free to reopen if I made a mistake.
/close

**Pull request closed without merging**